tests/BuiltInServer.php

```diff
@@ -108,14 +108,17 @@ public function start()
 
         // wait until it's listening for connections to avoid race conditions
         $start = microtime(true);
-        while (($sock = @fsockopen('localhost', $port, $errno, $errstr, 10)) === false) {
-            // set a 5 secs timeout waiting for the server to start
-            if (microtime(true) > $start + 5) {
-                $this->pid = false; // signal failure
-                $this->address = false;
-                break;
+        do {
+            $sock = @fsockopen('localhost', $port, $errno, $errstr, 10);
+            if ($sock === false) {
+                // set a 5 secs timeout waiting for the server to start
+                if (microtime(true) > $start + 5) {
+                    $this->pid = 0; // signal failure
+                    break;
+                }
             }
-        }
+        } while ($sock === false);
+
```

Comment:
I think Psalm complained about the $sock-variable.. Also removed `$this->address=false` because we already have `$this->pid` to signal a failure.. Makes the type-decl of `$this->address` more simple